hack/verify-gofmt.sh

```diff
@@ -21,8 +21,8 @@ cd "${OS_ROOT}"
 
 bad_files=$(find_files | xargs gofmt -s -l)
 if [[ -n "${bad_files}" ]]; then
-  echo "!!! gofmt needs to be run on the following files: "
+  >&2 echo "!!! gofmt needs to be run on the following files: "
```

Comment:
I'm also surprised by this being at the start of the line.  Didn't know that was a thing, can we move it to the end?

@stevekuznetsov He's doing this because pipes take stdout, not stderr.  This change allows `hack/verify-gofmt.sh | xargs gofmt -s -w` (which he should suggest in his output) to work.